.github/workflows/performance-benchmark.yml

```diff
@@ -0,0 +1,66 @@
+name: github-benchmarks
+on:
+  push:
+    branches:
+      - develop
+      - performance-results-visualization
```

Comment:
```suggestion
```

Reply:
This should only be triggered on push to `develop`.
Test runs can be observed on my branch, for example https://github.com/cz4rs/kokkos/actions/runs/3741965180.

Reply:
Another option would be to run the benchmarks on PRs also, just without pushing the results further.